README.md

```diff
@@ -1,4 +1,6 @@
 # `here`
+<!-- NOTE: This status badge is on nwtgck/here -->
+[![Build Status](https://travis-ci.org/nwtgck/here.svg?branch=master)](https://travis-ci.org/nwtgck/here)
```

Comment:
You can sign up Travis in https://travis-ci.org/. You can sign up Travis by your GitHub account. 

Then you can activate `tmhedberg/here` like the following image (✓ means activate).

![screen shot 2018-03-28 at 17 02 10](https://user-images.githubusercontent.com/10933561/38016390-cd0ce0fc-32a9-11e8-9770-f91e1284b1ba.png)

Then you can replace your README with
```md
[![Build Status](https://travis-ci.org/tmhedberg/here.svg?branch=master)](https://travis-ci.org/tmhedberg/here)
```
The image will be automatically updated by your build status.

If you want to trigger Travis build, you can push some commit to GitHub.

Reply:
I don't doubt that there are some advantages to using continuous integration in a project which is under active development and changing regularly. But `here` changes only occasionally. I don't think it's worth setting this up, and I don't intend to do it myself. Just having a test suite that we can run from the command line on demand is good enough.

Reply:
You can ignore `.travis.yml`. If you would merge the test suite, you can merge changes without `.travis.yml`.

I have the following three reasons why I added CI setting.

* The library will trust `here`  more
  - CI guarantees the library can be built on other environment than an author environment and, pass tests
* Test on different GHC versions (8.2, 8.0, 7.10, 7.8 and 7.6) ([a build result](https://travis-ci.org/))
  - The `.travis.yml` tests on the different versions. It will help the users to know supported GHC version.
* For future pull-requests
  - Pull-request will be automatically checked on Travis. It is easy for future contributors to know whether their request is valid or not.

> Just having a test suite that we can run from the command line on demand is good enough.

I also agree with your opinion. In addition, someone has to maintenance the CI setting. It takes some costs.